### DISCUSSION TITLE
refresh SpatieMediaLibraryFileUpload field

### DISCUSSION THREAD

**Discussion by cthutqc** [filamentphp/filament#5645] [General]
Hi

I make image field
SpatieMediaLibraryFileUpload::make('image')

After custom action pull
$this->refreshFormData([ 'name, text, image']);

name and text fields are updated, but image is disappear, and show only after refresh page
how refresh it too

**Comment by danharrin** [comment]
Maybe something like

```php
$this->data['image'] => $this->record->getMedia()
    ->mapWithKeys(function (Media $file): array {
        $uuid = $file->getAttributeValue('uuid');
    
        return [$uuid => $uuid];
    })
    ->toArray()
```
